From bcd4745460ebcfb23d7e10bb0261b68a89d6bc2f Mon Sep 17 00:00:00 2001
From: Christoph Berg <myon@debian.org>
Date: Sat, 22 Oct 2022 15:32:50 +0200
Subject: [PATCH] man/deb-src-control.pod: Clarify Build-Profiles syntax

Make it clear that "same syntax" for Build-Profiles includes the angle
brackets.
---
 man/deb-src-control.pod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/deb-src-control.pod b/man/deb-src-control.pod
index 7ca7ffb51..d3a593573 100644
--- a/man/deb-src-control.pod
+++ b/man/deb-src-control.pod
@@ -362,7 +362,7 @@ for more information about them).
 This field specifies the conditions for which this binary package does or
 does not build.
 To express that condition, the same restriction formula syntax from the
-B<Build-Depends> field is used.
+B<Build-Depends> field is used (including the angle brackets).
 
 If a binary package paragraph does not contain this field, then it implicitly
 means that it builds with all build profiles (including none at all).
-- 
2.35.1

